.github/workflows/repro_check.yml

```diff
@@ -0,0 +1,79 @@
+name: Build and Diff Projects
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
+      - reproducible
```

Comment:
```suggestion
```
If this job takes 4 hours we can't run it on pull requests because it would be too slow.